<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rst_main_icon</class>
 <widget class="QWidget" name="rst_main_icon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="4">
    <widget class="QLabel" name="rst_review">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>(1,000)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="rst_deliverytime">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>18~40분</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QLabel" name="rst_deliveryfee">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>배달비 1,000원</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="rst_star">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>5.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="rst_image_icon" native="true">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>110</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/header_img/AMERICANPIZZA.jpg);
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/ic_item_star_v2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QLabel" name="rst_name">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>아메리칸피자-남구점</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
